src/components/FormattedNumber/FormattedNumber.tsx

```diff
@@ -1,13 +1,67 @@
-import React, { useMemo, forwardRef } from "react";
-import PropTypes from "prop-types";
+import React, { useMemo, forwardRef, ForwardedRef } from "react";
 import classNames from "classnames";
 import { formatNumber, formatNumberConsts } from "../../helpers/textManipulations";
 import { baseClassName } from "./FormattedNumberConsts";
 import { validateValue } from "./FormattedNumberHelpers";
 import "./FormattedNumber.scss";
+import VibeComponentProps from "src/types/VibeComponentProps";
```

Comment:
great job, due to internal requirements please use the relative path 🙏

Reply:
Thanks for review. I've updated both imports to use relative path.